### ISSUE TITLE
Add "advertising annoyance level"

### ISSUE THREAD

**Issue by monroemann** [breakdiving/tosfairness#40]
Perhaps also rate the company on how annoying their advertising is.  Are there constant pop ups?  If it's a game, does it show you an ad every 2 minutes?

**Comment by taratip** [comment]
So we will have 16 Ratings dding up 100 in total. How many points for this rating?

**Comment by monroemann** [comment]
No, this is something we can add later.  It was just something I was thinking about.  :)
